### ISSUE TITLE
TypeError: ErrorOverlayPlugin is not a constructor

### ISSUE THREAD

**Issue by hozefaj** [smooth-code/error-overlay-webpack-plugin#3]
```
/Users/hjodiawalla/Desktop/CEP/personalizationnodeweb/webpack.config.js:11
    new ErrorOverlayPlugin(),
    ^

TypeError: ErrorOverlayPlugin is not a constructor
```

From the source code, looks like `ErrorOverlayPlugin` is a `class` and not a function constructor.

**Comment by neoziro** [comment]
Try to get `.default` As a workaround it should be a problem in export. I will fix it soon.

**Comment by billyjanitsch** [comment]
@neoziro thanks a lot for addressing this! However the fix was a breaking change and ideally it should've been released as 0.2.0 to avoid breaking existing builds that used the 0.1.2 API. Maybe consider deprecating 0.1.3 and re-releasing?

**Comment by neoziro** [comment]
You are right, sorry for that. I think the usage is still very moderate so I will not publish again.
